clients/include/flops.hpp

```diff
@@ -255,9 +255,10 @@ constexpr double symv_gflop_count(rocblas_int n)
 }
 
 /* \brief floating point counts of GER */
-template <typename T>
+template <typename T, bool CONJ>
 constexpr double ger_gflop_count(rocblas_int m, rocblas_int n)
 {
+    // conjugate not counted
```

Comment:
Need to have a different calculation for complex types.

Reply:
Thanks forgot, will add it.